src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+      ">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="Page">
+    <div class="Card" data-qa="card">
+      <figure class="Card-imgContainer">
+        <img
+          src="images/imac.jpeg"
+          alt="imac image"
+          class="Card-img"
+        >
+        <figcaption class="Card-imgDescription">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </figcaption>
+      </figure>
+      <p class="Card-code">Код товара: <span>195434</span></p>
+      <div class="Card-reviewContainer">
+        <div class="Card-reviewStars">
+          <span class="Card-star Card-star--active"></span>
+          <span class="Card-star Card-star--active"></span>
+          <span class="Card-star Card-star--active"></span>
+          <span class="Card-star Card-star--active"></span>
+          <span class="Card-star"></span>
+        </div>
```

Comment:
Let's use approach from `stars` task for styling stars here (nth-child with modificator)